nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -395,6 +399,29 @@ def simulator_run_main(self):
             run_status = 1
         return run_status
 
+    def split_gpus(self, gpus) -> ([str], bool):
```

Comment:
> why the function returns ( List[str], bool) ? instead of ( List[Tuple], bool) ?
> 
> essentially "[1,2],[3], [4,5,6]" --> split_gpus() -> [(1,2), (3,), (4,5,6)]

Just need to have the list of GPU IDs in string. I don't have the usage case for a Tuple.